public/tabs/chat/loading_placeholder.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { ReactNode, useEffect, useState } from 'react';
+import { MessageBubble } from './messages/message_bubble';
+
+interface LoadingPlaceholderProps {
+  loading: boolean;
+  height: number;
+  children?: ReactNode;
+}
+
+export const LoadingPlaceholder: React.FC<LoadingPlaceholderProps> = ({
+  loading,
+  height,
+  children,
+}: LoadingPlaceholderProps) => {
+  const [showLoadingPlaceholder, setShowLoadingPlaceholder] = useState(false);
```

Comment:
Seems using props.loading is good enough, do we need the state?

Reply:
Nope, I used to do other thing in the useEffect but then removed, seems this useEffect is not necessary entirely and I forgot to also remove it, thanks for suggestion.